pages/common/gcloud-sql-backups-describe.md

```diff
@@ -0,0 +1,8 @@
+# gcloud sql backups describe
+
+> Retrieves information about a backup.
+> More information: <https://cloud.google.com/sdk/gcloud/reference/sql/backups/describe>.
+
+- Code description:
```

Comment:
```suggestion
> Retrieve information about a backup.
> See also: `gcloud`.
> More information: <https://cloud.google.com/sdk/gcloud/reference/sql/backups/describe>.

- Retrieve information about a backup:
```

We can update the description to match the documentation and reference the main page.